.github/workflows/update-unified-manifest.yml

```diff
@@ -0,0 +1,41 @@
+name: build unified manifest file
+on: [push]
+jobs:
+  build_file:
+    name: build unified manifest file
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Run script
+        uses: jannekem/run-python-script-action@v1
+        with:
+          script: |
+            import os, json
+            with open("manifest.json", "r+") as t:
+              t.truncate(0)
+              t.write('[')
+              first = True
```

Comment:
didnt we find a json lib to parse this? i really think this is redundant and dangerous

Reply:
the only thing the json library is good for here is using `dumps` function to write the json to file, and it parse it with `/n`s ans `/t`s, so the normal text writer just works better...
couldn't find a solution specific enough merge multiple json files into an array, all the solutions i found simply run over the files and write them to the file one by one

Reply:
hmm and something like this doesnt owrk?
<img width="681" alt="image" src="https://user-images.githubusercontent.com/22911480/174757746-6a65eeaf-5cd6-4a8f-8504-2b2541eac3d1.png">